streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedTasks.java

```diff
@@ -110,7 +111,6 @@ boolean hasRunningTasks() {
     void transitionToRunning(final T task) {
         log.debug("Transitioning {} {} to running", taskTypeName, task.id());
         running.put(task.id(), task);
-        task.initializeTaskTime();
```

Comment:
we remove the method entirely and init task time in the constructor of `StreamTask` (note, that we don't need to init task time for `StandbyTasks` anyway)

Reply:
This in now done implicitly with `resume()`